server/hangouts/methods.js

```diff
@@ -88,6 +88,11 @@ Meteor.methods({
         id: group._id,
         title: group.title,
         slug: group.slug
+      },
+      email_notifications: {
+        initial: false,
+        reminder: false,
+        follow_up: false
```

Comment:
Does this mean users of a group won't receive email alerts when a hangout in that group is created?

Reply:
No, These flags are there so cron job can keep track for which notification has been sent.
we makes it true once notification is sent. [here](https://github.com/codebuddies/codebuddies/pull/801/files/8d65e59ddf86bfbc9e7a295b7252a85827ef7ff9#diff-6c4f4a0e79a2047c67141624ce02c65aR73)